### DISCUSSION TITLE
GET Webhook

### DISCUSSION THREAD

**Discussion by imknight** [directus/directus#15988] [Archive]
I know we can create a flow to handle the GET webhook, but will it be possible to redirect the user or display some process results after access the GET webhook?

**Comment by br41nslug** [comment]
How do you mean redirect the user? Webhooks are often triggered by external systems without direct user interaction 🤔 You can return data on the triggering call by setting the "Response Body" if "Asynchronous" is `off` that is.
![image](https://user-images.githubusercontent.com/9389634/195804815-15fe2ec4-0ecc-4875-9a6a-5679560edf50.png)

**Reply by imknight** [reply]
is there anyway to customize the response body ? like return "Status is ok" etc ?

**Reply by imknight** [reply]
found it, if anyway need this answer, you can just have a json transformer and put the key of that transformer here.
